README.md

```diff
@@ -97,6 +97,34 @@ CONVERSATION_PASSWORD=<add_conversation_password>
 WORKSPACE_ID=<add_conversation_workspace>
 ```
 
+### New API authentication process
```

Comment:
As a first time reader, I don't care about "New". I just tried to follow the pattern and got really lost looking at the above section.